docker/Dockerfile

```diff
@@ -31,9 +25,7 @@ RUN if [[ "${release}" == 1 ]] ; then \
         --static --warnings all \
         --link-flags "-lxml2 -llzma"; \
     fi
-
-
-FROM alpine:3.16
+FROM alpine:3.18
```

Comment:
Can you keep the double return to the line here?

```
[....]
        --link-flags "-lxml2 -llzma"; \
    fi


FROM alpine:3.18
[....]
```

Reply:
> Can you keep the double return to the line here?
> 
> ```
> [....]
>         --link-flags "-lxml2 -llzma"; \
>     fi
> 
> 
> FROM alpine:3.18
> [....]
> ```

Fixed!